assignment4/src/app/app.component.ts

```diff
@@ -6,5 +6,18 @@ import { Component } from '@angular/core';
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'assignment4';
+  oddnumbers: number[] = [];
+  evennumbers: number[] = [];
+ // tslint:disable-next-line: typedef
+ onGameStarted(gameTime: number ){
+  console.log('Also time is shown from app component ' + gameTime + 'sec' );
```

Comment:
Refrain from adding `console.log` statements into your commits. It is used for debugging, no point to have it in the main branch ;)